qiskit_finance/data_providers/exchange_data_provider.py

```diff
@@ -88,7 +88,7 @@ def __init__(
 
     def run(self) -> None:
         """
-        Loads data, thus enabling get_similarity_matrix and get_covariance_matrix
+        Loads data, thus enabling :code:`get_similarity_matrix` and :code:`get_covariance_matrix`
```

Comment:
I think this is ok, but preferably when referring to classes, functions, methods etc there are better to use as :code: simply highlights the text whereas using :meth: in this case where would also create the output as links to the methods. Note the way these end up documented they appear as method in the API ref in this class as it includes inherited methods so the fact that these may be in the base class one doesn't really need to know to use them.

Reply:
Thanks for suggesting it, I learnt a useful key binding!